packages/donations-app/app/components/dashboard/charityDashboardController.js

```diff
@@ -42,6 +43,8 @@ angular.module('aliceApp')
           });
           vm.projectValidator = vm.projectWithGoals.projectValidator;
         }
+
+        $timeout(()=> {$scope.dataLoaded = true;}, 3000);
```

Comment:
Why are you waiting for 3 secs?

Reply:
Time taken to finish our new loading animation

Reply:
Ok, thx for the clarification.